package.json

```diff
@@ -3,14 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "chart.js": "^2.9.3",
-    "react-bootstrap": "^1.0.0",
-    "react-chartjs-2": "^2.9.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "amazon-cognito-identity-js": "^4.2.1",
+    "aws-sdk": "^2.656.0",
     "bootstrap": "^4.4.1",
+    "chart.js": "^2.9.3",
```

Comment:
Do we need chart.js and react-chartjs-2?

Reply:
I was planning on using it for the Statistics section for dope circles, not needed rn though.